### PR TITLE
DOCS-1905: Update C++ code samples

### PR DESCRIPTION
**Pull request by npentrel**

### CODE REVIEW

File: docs/build/program/_index.md [hunk 1]
```diff
@@ -69,7 +69,9 @@ The TypeScript SDK currently only supports building web browser apps.
 {{% /tab %}}
 {{% tab name="C++" %}}
 
-Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+```sh {class="command-line" data-prompt="$"}
+brew install viam-cpp-sdk
```
[the remark this answers is not in the document]
Comment (stuqdog) [reply docs/build/program/_index.md:73]: I think it's likely that more work is needed to support `brew install`. At a minimum we'll need to inform folks on the best way to actually find the SDK, but I don't think we have good `find_package` support currently. For now can we stick with the manual build? I can start digging into this and will be happy to update once I've done so.

**Review by stuqdog — APPROVED** [review]
Other than wanting to stick with the manual build steps for now this lgtm!

File: docs/build/program/_index.md (hunk 1 above)
Comment (npentrel) [inline docs/build/program/_index.md:73]:
```suggestion
Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
```

File: docs/build/program/_index.md [hunk 2]
```diff
@@ -69,7 +69,9 @@ The TypeScript SDK currently only supports building web browser apps.
 {{% /tab %}}
 {{% tab name="C++" %}}
 
+```sh {class="command-line" data-prompt="$"}
 Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
+```
```
Comment (npentrel) [inline docs/build/program/_index.md:74]:
```suggestion
Follow the [instructions on the GitHub repository](https://github.com/viamrobotics/viam-cpp-sdk/blob/main/BUILDING.md).
```

**Pull request merged**